src/Microsoft.PowerShell.Commands.Diagnostics/PdhHelper.cs

```diff
@@ -267,28 +267,65 @@ private struct PDH_TIME_INFO
         // We only need dwType and lDefaultScale fields from this structure.
         // We access those fields directly. The struct is here for reference only.
         //
-        [StructLayout(LayoutKind.Explicit, CharSet = CharSet.Unicode)]
-        private struct PDH_COUNTER_INFO
+        [StructLayout(LayoutKind.Sequential)]
```

Comment:
Do we need CharSet?

Reply:
I can add it if preferred but the struct is directly blittable so it's not strictly necessary.

Reply:
If not mandatory, we should not have it.